XpringKit.xcodeproj/project.pbxproj

```diff
@@ -701,6 +701,8 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 1020;
+				TargetAttributes = {
```

Comment:
This is a generated file, I think that either my local version of XCode / the generation tool were updated. This should be a no-op.